src/festim/hydrogen_transport_problem.py

```diff
@@ -818,6 +827,17 @@ def post_processing(self):
                 # if filename given write export data to file
                 if export.filename is not None:
                     export.write(t=float(self.t))
+
+            elif isinstance(export, exports.SurfaceTemperature):
+                export.compute(self.ds)  # compute surface temp
+
+                export.t.append(float(self.t))  # update export time
+
+                # if filename given write export data to file
+                if export.filename is not None:
+                    export.write(t=float(self.t))  
```

Comment:
These lines are not tested. That's because by design, it's impossible to use `SurfaceTemperature` in combination with `HTransportProblem`